src/main/kotlin/com/github/pool_party/pull_party_bot/Main.kt

```diff
@@ -0,0 +1,32 @@
+package com.github.pool_party.pull_party_bot
+
+import com.elbekD.bot.Bot
+import com.elbekD.bot.server
+import com.elbekD.bot.util.AllowedUpdate
+import com.github.pool_party.pull_party_bot.commands.initializePingCommands
+
+const val APP_URL = "https://somebodyoncetoldmepool.herokuapp.com"
+const val USER_NAME = "PullPartyBot"
+const val DEFAULT_PORT = 80
+
+fun main() {
+    val token = System.getenv("TELEGRAM_TOKEN") ?: throw RuntimeException("Unable to get system variable for token")
+    val isLongpoll = System.getenv("IS_LONGPOLL")?.toBoolean() ?: false
```

Comment:
Let's consider making these environment variables command line flags in the future

Reply:
We will rework all system-variables-things in a future anyway

Reply:
I'll try 2 mak'em flags l8r, that seems more convenient 4 me. Anyway, we'll discuss l8r, as well